build/pack.ps1

```diff
@@ -7,36 +7,28 @@
 #>
 
 param (
-  [string[]] $PackageDirs,
-  [string[]] $EnvNames,
+  [string] $PackageName,
   [string] $OutDir
 )
 
 & (Join-Path $PSScriptRoot "set-env.ps1");
 
 Import-Module (Join-Path $PSScriptRoot "conda-utils.psm1");
 
-if ($null -eq $PackageDirs) {
+if ('' -eq $PackageName) {
```

Comment:
looks like this code is a great candidate to move to conda-utils.psm1

Reply:
Moved to package-utils.psm1.